app/mailers/transaction_mailer.rb

```diff
@@ -83,6 +84,10 @@ def braintree_new_payment(payment, community)
     service_fee = payment.total_commission
     you_get = payment.seller_gets
 
+    listing = payment.transaction.listing
+    unit_type = translate_quantity_unit(listing.unit_type)
```

Comment:
It's a bad sign that we need to refer to listing here. We must record unit type to transaction because:
1. This mailer is transaction mailer and logically part of "transaction service", so reaching out to other services for data is a no no.
2. The unit type might already be different for listing at this point than when the transaction was started.